### PR TITLE
Fix/User status inconsistencies

### PR DESCRIPTION
**Pull request by yurii-lubynets**
closes https://github.com/sparkletown/internal-sparkle-issues/issues/1108

### CODE REVIEW

File: src/components/molecules/UserStatusManager/components/UserStatusPanel/UserStatusPanel.scss [hunk 1]
```diff
@@ -21,6 +21,7 @@ $status--spacing: $icon--spacing 0;
     border-radius: $border-radius--max;
     width: $picker-icon--size;
     height: $picker-icon--size;
+    margin-left: $icon--spacing;
```
Comment (yurii-lubynets) [inline src/components/molecules/UserStatusManager/components/UserStatusPanel/UserStatusPanel.scss:24]: fixed buttons indents

Screenshot 2021-09-29 at 15 40 37

File: src/pages/Admin/Venue/VenueDetailsSubForm.tsx [hunk 2]
```diff
@@ -727,6 +727,8 @@ export const VenueDetailsSubForm: React.FC<VenueDetailsSubFormProps> = ({
 
         {renderRadioToggle()}
 
+        {values.showRadio && renderRadioStationInput()}
```
Comment (yurii-lubynets) [inline src/pages/Admin/Venue/VenueDetailsSubForm.tsx:730]: fixed radio URL input based on comment https://github.com/sparkletown/internal-sparkle-issues/issues/1108#issue-979103157

File: src/components/organisms/NewProfileModal/components/header/ProfileModalAvatar/ProfileModalAvatar.tsx [hunk 3]
```diff
@@ -79,7 +76,7 @@ export const ProfileModalAvatar: React.FC<ProfileModalAvatarProps> = ({
           imageClassName="ProfileModalAvatar__image"
           user={userWithOverriddenPictureUrl}
           size="full"
-          showStatus={!isCurrentUser}
+          showStatus
```
Comment (yurii-lubynets) [inline src/components/organisms/NewProfileModal/components/header/ProfileModalAvatar/ProfileModalAvatar.tsx:79]: fixed status displaying for edit window based on https://github.com/sparkletown/internal-sparkle-issues/issues/1108#issue-979103157

**Review by goran-peoski-work — COMMENTED** [review]
Code looks as expected.

Functionality is there in Av1: changing the status and updating the venue in does update statuses for users.  

Missing in Av3: Can't determine from issue if by design, unintentional omission or plan to fix in next PR as part of world settings
 
sparkle-show-user-status-01

**Review by goran-peoski-work — APPROVED** [review]
Found the change for Av3 🥳 
sparkle-show-user-status-03

File: src/components/atoms/UserAvatar/UserAvatar.tsx [hunk 4]
```diff
@@ -106,7 +106,8 @@ export const _UserAvatar: React.FC<UserAvatarProps> = ({
   //'showStatus' is used to render this conditionally only in some of the screens.
   const hasUserStatus =
     isStatusEnabledForVenue &&
-    isOnline &&
+    // @debt temporarily disable is online functionality
```
[the remark this answers is not in the document]
Comment (yurii-lubynets) [reply src/components/atoms/UserAvatar/UserAvatar.tsx:109]: fixed

**Pull request merged**